api_app/services/cidr_service.py

```diff
@@ -1,5 +1,6 @@
 from typing import List
 from ipaddress import IPv4Network, NetmaskValueError, collapse_addresses
+from xmlrpc.client import boolean
```

Comment:
I don't think you want this (see other comment)

Reply:
Do you know - I dont even know where that came from!